### DISCUSSION TITLE
Audi A4 B9  FEC data error？

### DISCUSSION THREAD

**Discussion by Lanye-z** [Mr-MIBonk/M.I.B._More-Incredible-Bash#100] [Q&A] [answered]
hello ，I followed the procedure to the letter,but when I check the fec I don't have the correct one 023200ee for unlimited maps but only the one there, 02320044. 
![Screenshot_20230203-215518_Samsung Internet](https://user-images.githubusercontent.com/80017187/216623758-a32b25dc-7dc8-4393-b74e-3933ee7603be.jpg)

Through comparing the documents, I found some problems.
The addFecs in “\M.I.B_Patches_20220625\addFec\Audi\CN” have the 023200ee，
![22](https://user-images.githubusercontent.com/80017187/216625034-6ce18a5d-718f-4bae-b683-dce251a63d11.png)

but the  addFecs  in “\M.I.B_Patches_20220625\MHI2Q_CN_AUG22_P1002_MU1002_PATCH” only have the 023020044

![21](https://user-images.githubusercontent.com/80017187/216625023-36f325be-bffa-477d-bdbc-46a08bbdd744.png)

What should i do to have the correct fec?
Change the addfecs  in “\M.I.B_Patches_20220625\MHI2Q_CN_AUG22_P1002_MU1002_PATCH” from 023020044 to 023200ee，and try again？

**Comment by Lanye-z** [comment, marked as answer]
meanwhile，here is a picture  from the Internet，023200ff is also legal?
![970_wKjByVo8YlqAUQSgAAEiH4Me4hg227](https://user-images.githubusercontent.com/80017187/216643820-18ced93d-d078-4259-b617-fc06b41770c5.jpg)

**Reply by harman-f** [reply]
023200ff can lead to issues with your unit - do not use it.

02320044 should also be fine for latest maps.
just edit addFec.txt to 023200ee and run addFec function in GEM again.

**Reply by Lanye-z** [reply]
Thank you very much
